### PR TITLE
[13.0][MIG]  hr_expense_sequence

### PR DESCRIPTION
**Pull request by ps-tubtim**

### CODE REVIEW

File: hr_expense_sequence/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "HR expense sequence",
+    "version": "13.0.1.0.0",
+    "license": "AGPL-3",
+    "category": "Human Resources",
+    "author": "Serv. Tecnol. Avanzados - Pedro M. Baeza,"
+    "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/hr",
```
Comment (Saran440) [inline hr_expense_sequence/__manifest__.py]:
```suggestion
    "website": "https://github.com/OCA/hr-expense",
```

Comment (ps-tubtim) [reply hr_expense_sequence/__manifest__.py]: @Saran440 Thanks for review. Edited.

**Review by Saran440 — APPROVED** [review]
LGTM :+1:

**Comment by georginaf** [comment]
Functional LGTM

File: hr_expense_sequence/data/hr_expense_data.xml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+    <record model="ir.sequence" id="seq_expense">
```
Comment (Saran440) [inline hr_expense_sequence/data/hr_expense_data.xml]:
```suggestion
    <record id="seq_expense" model="ir.sequence">
```

Comment (ps-tubtim) [reply hr_expense_sequence/data/hr_expense_data.xml]: Thank you for your review. I edited.

**Comment by ps-tubtim** [comment]
@newtratip @kittiu Could you review this PR?

**Review by kittiu — APPROVED** [review]

**Pull request merged**